components/FixedBanner.vue

```diff
@@ -37,12 +42,38 @@ export default {
       };
     },
 
+    bannerConsentStyle() {
```

Comment:
@jordojordo This still duplicates the code above - where I think you should get to is a FixedBanner that has three modes - header, footer and consent. As is, a fixed banner can show a fixed banner + optionally the consent banner - I'd changed that so FixedBanner only shows one type of banner - then in the unauthenticated layout, you would have 3 FixedBanner components - one for header, consent and footer. This would reduce duplicated code.

Reply:
@nwmac I modified FixedBanner to have the three modes - `header`, `consent`, and `footer`. Depending on the type passed it will now show the correct banner. When importing FixedBanner there is now a prop for each type, one of these must be set for the corresponding banner to show.